include/roboptim/core/manifold-map/decorator/manifold-problem-factory.hh

```diff
@@ -83,6 +83,12 @@ namespace roboptim
     template<class V, class W>
     void setObjective(DescriptiveWrapper<V, W>& descWrap, mnf::Manifold& instanceManifold);
 
+    /// \brief for a given elementary manifold, add bounds to its arguments
+    ///
+    /// \param manifold the elementary manifold to bound
```

Comment:
Bah, my bad indeed, that will teach me to check a single only dictionary a bit too quickly! All good then ;-)